src/resize.rs

```diff
@@ -116,6 +106,19 @@ pub enum InterpolationMode {
     Nearest,
 }
 
+pub(crate) fn interpolate(
```

Comment:
```suggestion
pub(crate) fn interpolate_pixel(
```
I’m trying to avoid high level names for names. Generic apis names will come later once everything is mature